### DISCUSSION TITLE
Using a single postgres instance for all the different apps

### DISCUSSION THREAD

**Discussion by minhajuddin** [dokku/dokku#5118] [General]
Hey Guys, I am just getting started with Dokku and I love it, it has made things so much easier to manage. I have a big bare metal server with 64GB with 6 cores and 12 threads. Are there any blog posts or setup where there is a single Postgres instance running on the host which serves all the other web apps? similar to how nginx fronts all the traffic to the apps. Has anyone done any perf benchmarking around these 2 alternatives?

**Comment by josegonzalez** [comment]
There isnt anything special about that other than routing to the host from a container. [This answer](https://stackoverflow.com/a/43541732) on stackoverflow may be helpful for setting a hostname for your containers to route to the host.

As far as performance is concerned, what are you trying to compare it to?
